### ISSUE TITLE
Named export error 

### ISSUE THREAD

**Issue by wcardosodev** [anubra266/choc-autocomplete#241]
Getting the below error on version "v5.2.9"

SyntaxError: Named export 'AutoComplete' not found. The requested module '@choc-ui/chakra-autocomplete' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@choc-ui/chakra-autocomplete';

**Comment by Kysluss** [comment]
Hello @wcardosodev 

Would you be able to post a repro of this issue?  I'm not able to reproduce it.

**Comment by omniverse** [comment]
I've had the same issue.
I'm using choc-autocomplete in a Remix project with TS. `"type": "module"` in my package.json file. 
My workaround is to add a postinstall script that modifies the choc module's `type` setting in its package.json after install. Then the Remix build does not complain.

```
// inspired by https://github.com/ds300/patch-package (but it doesn't work with workspaces yet)
import fs from 'fs';

const cwd = process.cwd();
const moduleFixes = [
  `${cwd}/node_modules/@choc-ui/chakra-autocomplete/package.json`
];

try {
  moduleFixes.forEach((path) => {
    const packageString = fs.readFileSync(path, { encoding: 'utf8' });
    const packageObject = JSON.parse(packageString);
    fs.writeFileSync(
      path,
      JSON.stringify({ type: 'module', ...packageObject }, null, 2),
    );
    console.log(`Patched ${path}`);
  });
} catch (err) {
  console.log(err);
  process.exit(1);
}
```

**Comment by Kysluss** [comment]
Hi @omniverse 

I'm not very familiar with Remix, but if you're able to post a simple repo with your fix in it, I'll try to remove it so it reproduces the error and see if I can fix it.

**Comment by ImExoOdeex** [comment]
i'm having same issue using remix only with vite. The vite compiles the code successfully, but when trying to run the build server file, it crashes and says:
```
import { AutoComplete, AutoCompleteInput, AutoCompleteTag, AutoCompleteList, AutoCompleteItem, AutoCompleteCreatable } from "@choc-ui/chakra-autocomplete";
         ^^^^^^^^^^^^
SyntaxError: The requested module '@choc-ui/chakra-autocomplete' does not provide an export named 'AutoComplete'  
    at ModuleJob._instantiate (node:internal/modules/esm/module_job:123:21)
    at ModuleJob.run (node:internal/modules/esm/module_job:189:5)
```

However, @omniverse's fix works fine

**Comment by Kysluss** [comment]
@ImExoOdeex would you be able to post up a repo containing the issue or steps on how you're able to reproduce it?  I'd really like to get this fixed, but I'm having trouble reproducing it.  I'm glad that there's a workaround available for the time being.

**Comment by Kysluss** [comment]
Thank you all for your patience @wcardosodev @omniverse and @ImExoOdeex .  I was finally able to take some time and work up a repro of this issue.  It looks like the issue you were seeing was due to a problem with the build setup we had.  This should be fixed in [v5.4.0](https://github.com/anubra266/choc-autocomplete/releases/tag/v5.4.0)
